instrumentation/graphql/test/test_helper.rb

```diff
@@ -125,3 +125,12 @@ def uses_platform_interfaces?
 def gem_version
   Gem::Version.new(GraphQL::VERSION)
 end
+
+# When tracing, is the parser expected to call `lex` before `parse`
+def trace_lex_supported?
+  return @trace_lex_supported if defined?(@trace_lex_supported)
+
+  # In GraphQL 2.2, the default parser was changed such that `lex` is no longer called
+  @trace_lex_supported = Gem::Requirement.new('< 2.2').satisfied_by?(Gem::Version.new(GraphQL::VERSION)) \
+    || (defined?(GraphQL::CParser) == 'constant')
```

Comment:
Does anything result in `require "graphql/c_parser"` such that this is `true` when the gem is installed?

Reply:
The test helper uses bundler require, which will load all of the declared gems by default before running any tests.

In this case the Gemfile requires the graphql-c_parser file in the gems' lib directory, which requires `graphql/c_parser`... then turtles all the way down.

Reply:
https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/instrumentation/graphql/test/test_helper.rb#L8